Makefile

```diff
@@ -45,6 +44,9 @@ install-deps:
 
 # run golangci-lint
 lint:
+	golint .
+	golint ./discovery
```

Comment:
I want to add golint to golangci-lint config. Then it could be configured as other linters and golangci-lint does linters faster then outside one by one.

Reply:
I've done it in #127. @g4s8 please check. Maybe we can merge that PR first, and then revert changes in Makefile here?

Reply:
Or at first merge this PR and then I move golint to golangci-lint config and fix other issues around the project.

Reply:
@stepanvalyavskiy I've removed these commands from `lint` step